### PR TITLE
feat: locale-string added for rest locales

### PR DESCRIPTION
**Pull request by PritishBudhiraja**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Added locale support for -
```
  English (en)
  Hebrew (he)
  French (fr)
  English (en-GB)
  Arabic (ar)
  Japanese (ja)
  German (de)
  French (Belgium) (fr-BE)
  Spanish (es)
  Catalan (ca)
  Portuguese (pt)
  Italian (it)
  Polish (pl)
  Dutch (nl)
  Swedish (SV)
  Russian (ru)

```

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/LocaleString.res [hunk 1]
```diff
@@ -717,7 +718,739 @@ let localeStrings = [
...
   },
+  {
+    locale: "fr-BE",
+    localeDirection: "ltr",
+    cardNumberLabel: "Numéro de carte",
+    inValidCardErrorText: "Le numéro de carte n'est pas valide.",
+    inCompleteCVCErrorText: "Le code de sécurité de votre carte est incomplet.",
+    inCompleteExpiryErrorText: "La date d'expiration de votre carte est incomplète.",
+    pastExpiryErrorText: "L'année d'expiration de votre carte est passée.",
+    poweredBy: "Powered By Hyperswitch",
```
Comment (vsrivatsa-juspay) [inline src/LocaleString.res:732]: Is this intentionally in English?

**Review by prafulkoppalkar — APPROVED** [review]

File: src/LocaleString.res (hunk 1 above)
Comment (prafulkoppalkar) [reply src/LocaleString.res:741]: yes

File: src/LocaleString.res [hunk 2]
```diff
@@ -111,9 +111,9 @@ let defaultLocale = {
   useExisitingSavedCards: "Use saved debit/credit cards",
   saveCardDetails: "Save card details",
   addBankAccount: "Add bank account",
-  achBankDebitTerms: _str =>
+  achBankDebitTerms: _ =>
     `Your ACH Debit Authorization will be set up now, but we'll confirm the amount and let you know before future payments are taken.`,
-  sepaDebitTerms: _str =>
+  sepaDebitTerms: _ =>
```
Comment (ArushKapoorJuspay) [inline src/LocaleString.res:116]: If the argument is not being used, please remove it altogether

Comment (PritishBudhiraja) [reply src/LocaleString.res:116]: Updated

**Review by prafulkoppalkar — APPROVED** [review]

**Review by vsrivatsa-juspay — APPROVED** [review]

**Review by prafulkoppalkar — APPROVED** [review]

**Pull request merged**